packages/core/README.md

```diff
@@ -111,9 +112,10 @@ await percy.stop()
 
 ## Advanced
 
-### Puppeteer Executable Path
+### Chromium Executable Path
 
-To avoid downloading the browser used for asset discovery, the local browser executable can be
-defined with an `executablePath` option provided within `discovery.launchOptions`. This options
-falls back to the `PUPPETEER_EXECUTABLE_PATH` environment variable if defined.
+**Use with caution as asset discovery may not work with some versions of Chromium.** To avoid
```

Comment:
🔥 I see what you mean. We can update this then: https://docs.percy.io/docs/skipping-puppeteer-chromium-download